.github/workflows/on-prerelease.yml

```diff
@@ -60,10 +60,7 @@ jobs:
           ./build.ps1 -Command BuildAndPublish `
               -Configuration Release `
               -APIVersion $apiVersion
-
-      - name: Populate Google Analytics ID
-        run: .\build.ps1 -Command PopulateGoogleAnalyticsAppSettings -GoogleAnalyticsMeasurementId ${{env.GA_MEASUREMENT_ID}}
-
```

Comment:
I am assuming that we don't use the Google Analytics ID for Admin API, though I might be wrong and I might have been too aggressive.

Reply:
Good catch and good to remove.  You are right, we don't use Google Analytics anymore and definitely not in Admin API.